src/components/layouts/fullLayout.tsx

```diff
@@ -2,7 +2,7 @@ import { Outlet } from "react-router-dom";
 
 const FullLayout = (): JSX.Element => {
   return (
-    <div style={{ background: "black" }}>
+    <div style={{ background: "#a58a8a" }}>
```

Comment:
Do not use inline style: add sass for this instead

Reply:
ok used tail wind